bibliopixel/drivers/SimPixel/websocket.py

```diff
@@ -69,7 +69,7 @@ def target(self):
 
     def update(self, **state):
         self.state.update(state)
-        for client in self.clients:
+        for client in self.clients.copy():
```

Comment:
This is dubious to me... wouldn't this now update the copy, not the original?

Reply:
The copy only has scope in _that single line_.  

So a copy is made of `self.clients` before the loop starts, as a brand-new `set`.  During the iteration, `self.clients` might get updated in another thread, but that's fine because we're iterating over a copy.